modules/core/src/main/java/org/apache/ignite/internal/IgniteKernal.java

```diff
@@ -293,52 +294,79 @@
 import static org.apache.ignite.lifecycle.LifecycleEventType.BEFORE_NODE_START;
 
 /**
- * Ignite kernal.
- * <p/>
- * See <a href="http://en.wikipedia.org/wiki/Kernal">http://en.wikipedia.org/wiki/Kernal</a> for information on the
- * misspelling.
+ * This class represents an implementation of the extended main Ignite API (called {@link IgniteEx}). It also controls
```

Comment:
I think it worth to have a link to `Ignite` interface here as the current class is the real implementation of it. `IgniteEx` could be mentioned as well with a note that it adds several methods to `Ignite` for internal purposes. What do you think?

Reply:
Sound good. I've updated the class javadoc. Can you check?